src/components/projectconfig/candevicemodel.cpp

```diff
@@ -17,7 +17,12 @@ CanDeviceModel::CanDeviceModel()
     _name = "CanDeviceModel";
     _modelName = "CAN device";
 
-    _component.init("socketcan", "can0"); // TODO
+    QObject qo;
+    qo.setProperty("backend", "socketcan");
```

Comment:
Is there a chance to read this settings from the cds config file?

Reply:
Everything is possible, but this will require some refactor, as CanDeviceModel is currently instantiated when you create new project, so this acts as some default values I suppose (although honestly I'm not sure why it's called at all if there are no nodes in the project yet)